docs/getting-started.md

```diff
@@ -63,26 +69,26 @@ docker-compose build
 
 ## Adding dependencies
 
-- Get into web container bash
+- Get into server container bash
 
     ```bash
-    docker-compose exec web bash
+    docker-compose exec server bash
     ```
 
-- Adding Frontend Dependencies [JS]
+- Adding Client Dependencies [JS]
```

Comment:
We cannot do this inside server container but this step is after getting into the server container.
Probably need to fix to avoid confusion.

Reply:
Fixed!